go.mod

```diff
@@ -6,7 +6,7 @@
 
 module k8s.io/kubernetes
 
-go 1.16
+go 1.18
```

Comment:
this change has bigger implications, leave this at go1.16 in this PR

Reply:
Ok, I'm testing and I found a version that passes the golangci-lint test, but not the pull-kubernetes-dependencies

Do you have a good suggestion? `golang.org/x/sys` posted an incompatible tag